tools/spectrast/macros.xml

```diff
@@ -0,0 +1,15 @@
+<macros>
+    <token name="@VERSION@">5.0.0</token>
+    <xml name="requirements">
+        <requirements>
+        <requirement type="package">tpp=5.0.0</requirement>
```

Comment:
that looks wired :) you mean `<requirement type="package" version="5.0.0">tpp</requirement>`